test_data.rb

```diff
@@ -125,7 +125,7 @@ def test_unique_ids
         end
       end
 
-      bad_counts = counts.select { |station, count| !["LYON-", "MONTP", "NIMES"].include?(station) && count != 1 }
```

Comment:
Thank you, indeed the test is here for a reason, we should not have duplicate carrier IDs for the stations